product_barcode_generator/models/product.py

```diff
@@ -36,21 +36,20 @@ class ProductCategory(models.Model):
 class ProductProduct(models.Model):
     _inherit = 'product.product'
 
-    ean13 = fields.Char(copy=False)
     ean_sequence_id = fields.Many2one('ir.sequence', string='Ean sequence')
```

Comment:
Each time we migrate a module, we can (and should) improve the functionality, because this is the best chance of rethinking things.